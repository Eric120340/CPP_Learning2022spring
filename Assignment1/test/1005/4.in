3 3 1539929
35984606 247805894 39100324 
809128237 292710062 719646507 
284201311 346025531 536498042 
