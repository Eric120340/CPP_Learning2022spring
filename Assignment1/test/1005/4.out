245699231 267109864 678753182 
342778340 208501455 567304068 
363184928 630580145 831509354 
